test/Tests/src/Semantic/Case_Spec.enso

```
@@ -10,61 +17,86 @@ spec = Test.group "Pattern Matches" <|
             Boolean -> Nothing
             _ -> Test.fail "Expected the Boolean constructor to match."
         case True of
-            Boolean -> Nothing
-            _ -> Test.fail "Expected the True constructor to match."
+            Boolean -> Test.fail "Expected the Boolean type to match."
```

This failure message is not clear to me.

Maybe `fail "Instance of Boolean type unexpectedly matched the Boolean type-as-a-value."`?

---

`True != Boolean`. That's what we wanted. I agree that _"Expected ... to match"_ description that references the next line is a weird to read. Assuming all readers process the text from top to bottom, the prose section shall not reference to future lines.

---

I pretty much followed the style used in the rest of the spec. I can adjust it a bit if you want